docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -56,17 +122,25 @@ The `line_item` object has many common fields for browser events, including the
 |:---|:---|:---|:---|
 | `base_price` | number | Price of product. Default price in the control panel. The price should include or exclude tax, based on the store settings. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.price` |
 | `brand_name` | string | Brand name. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.brand_name` |
-| `category_names[]` | array | Category names. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.categories[]` |
+| `category_names[]` | array of strings | Category names. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.categories[]` |
 | `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) for the transaction. | [Get a checkout](/docs/rest-management/checkouts#get-a-checkout) <br />  `data.cart.currency.code`  |
-| `discount` | number | Discount applied to purchase. | [Get a checkout](/docs/rest-management/checkouts#get-a-checkout) <br /> `data.cart.discount_amount` |
-| `product_id` | string | ID of the product. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.id` |
+| `discount` | number | Discount applied to the item. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `data.cart.discountAmount` |
+| <sup>+</sup>`gift_certificate_id` | string | ID of gift certificate. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.id` |
+| <sup>+</sup>`gift_certificate_name` | string | Name of gift certificate that appears in the control panel. | [Get a Cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.name` |
+| <sup>+</sup>`gift_certificate_theme` | string | Theme of gift certificate. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.giftCertificates.theme` |
+| <sup>+</sup>`index` | integer | The position of the item in the cart. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br /> `data.cart.lineItems.physicalItems.id` or `data.cart.lineItems.digitalItems.id` |
+| `product_id` | integer | ID of the product. | [Get a product](/docs/rest-management/catalog/products#get-a-product) <br /> `data.id` |
```

Comment:
@bc-sanjayjoshi,
Type is conflicting with [the doc](https://docs.google.com/document/d/13qlYa9gtuVcjs2iBZ-jWP74W1GPRQw9J496t7QYbbJg/edit#heading=h.tbdia9ynarrk), where it's a string. However, it's in fact integer, so I think, the internal doc should be updated.

Reply:
@bc-sanjayjoshi This was about the `product_id` field. 

It seems like updating the internal doc would be the best fix. I'll go ahead and resolve the conversation so I can publish my doc